### DISCUSSION TITLE
Will it support Nuxt.js?

### DISCUSSION THREAD

**Discussion by rrmesquita** [wpengine/faustjs#574] [Ideas]
I'm really excited about this repo and what it's capable of, but I'm a Nuxt.js user so I wonder if Faust.js will ever support it. I'm looking at the code and everything seems to fit only React environments. If someone could please explain how the frontend pieces are connected together (something like a flowchart but written?), maybe I could try to make it work on Nuxt, as the backend code is pretty straightforward. Thanks!

**Comment by Saggitarie** [comment]
Also trying to implement this for my NuxtJS app? Is this WIP?

@rrmesquita Do you know any news about this? Thanks!

**Comment by blakewilson** [comment]
As of right now, we are focusing primarily on Next.js/React, and have no roadmap for NuxtJS support. However, you should be able to use Faust.js with NuxtJS via the [`@faustjs/core`](https://www.npmjs.com/package/@faustjs/core) package.

**Comment by Saggitarie** [comment]
@blakewilson 
Thanks for the notice and great work!

@faustjs/core works for post previews too then?

Also are there any documentation for using this?

**Comment by Saggitarie** [comment]
For those encountering the same issue.

Yes, @faustjs/core supports post previews and this still needs to use graphql.
I'm going to explore some more, hopefully get some solutions to it.

**Comment by woodyrew** [comment]
Did anyone find any documenation for using @faustjs/core? An how it might be used for other headless solutions?
